src/main/java/taxi/dao/impl/CarDaoImpl.java

```diff
@@ -211,10 +213,14 @@ private Driver parseDriverFromResultSet(ResultSet resultSet) throws SQLException
         Long driverId = resultSet.getObject("id", Long.class);
         String name = resultSet.getString("name");
         String licenseNumber = resultSet.getString("license_number");
+        String login = resultSet.getString("login");
+        String password = resultSet.getString("password");
```

Comment:
"id" - not very informative, better "driver_id"

Reply:
such naming was by default in project.
I`m not sure, but IDEA show me that its column label from table)